js/util/browser/dom.js

```diff
@@ -35,3 +37,15 @@ var transformProp = testProp(['transform', 'WebkitTransform']);
 exports.setTransform = function(el, value) {
     el.style[transformProp] = value;
 };
+
+exports.enableTextSelection = function() {
```

Comment:
These functions aren't necessary; use `disableDrag`/`enableDrag` instead, which accomplish the same thing using `user-select` CSS support.